src/vos/vos_internal.h

```diff
@@ -1072,7 +1072,8 @@ enum {
 	VOS_SLAB_EVT_NODE	= 3,
 	VOS_SLAB_EVT_DESC	= 4,
 	VOS_SLAB_OBJ_DF		= 5,
-	VOS_SLAB_MAX		= 6
+	VOS_SLAB_EVT_NODE_SM	= 6,
```

Comment:
Not a big deal, just don't see why this intermediate node is called 'NODE_SM'?

Reply:
It should probably be INT and LEAF but I chose SM for “small”